org.eclipse.Committers.json

```diff
@@ -0,0 +1,52 @@
+{
+  "id" : "org.eclipse.Committers",
+  "branch" : "4.13",
+  "runtime" : "org.gnome.Sdk",
+  "runtime-version" : "3.34",
+  "sdk" : "org.gnome.Sdk",
+  "sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk11" ],
+  "command" : "eclipse",
+  "modules" : [ {
+    "name" : "openjdk",
+    "buildsystem" : "simple",
+    "build-commands" : [ "/usr/lib/sdk/openjdk11/installjdk.sh" ]
+  }, {
+    "name" : "eclipse",
+    "buildsystem" : "simple",
+    "build-commands" : [
+      "mv eclipse /app",
+      "mkdir -p /app/bin",
+      "ln -s /app/eclipse/eclipse /app/bin",
+      "install -Dm 644 org.eclipse.Committers.desktop /app/share/applications/org.eclipse.Committers.desktop",
+      "install -Dm 644 org.eclipse.Committers.appdata.xml /app/share/metainfo/org.eclipse.Committers.appdata.xml",
+      "mkdir -p /app/share/icons/hicolor/32x32/apps",
+      "if [ -f /app/eclipse/plugins/org.eclipse.epp.package.committers_*/eclipse32.png ] ; then cp -p /app/eclipse/plugins/org.eclipse.epp.package.committers_*/eclipse32.png /app/share/icons/hicolor/32x32/apps/org.eclipse.Committers.png ; fi",
+      "mkdir -p /app/share/icons/hicolor/64x64/apps",
+      "if [ -f /app/eclipse/plugins/org.eclipse.epp.package.committers_*/eclipse64.png ] ; then cp -p /app/eclipse/plugins/org.eclipse.epp.package.committers_*/eclipse64.png /app/share/icons/hicolor/64x64/apps/org.eclipse.Committers.png ; fi",
+      "mkdir -p /app/share/icons/hicolor/128x128/apps",
+      "if [ -f /app/eclipse/plugins/org.eclipse.epp.package.committers_*/eclipse128.png ] ; then cp -p /app/eclipse/plugins/org.eclipse.epp.package.committers_*/eclipse128.png /app/share/icons/hicolor/128x128/apps/org.eclipse.Committers.png ; fi",
+      "mkdir -p /app/share/icons/hicolor/256x256/apps",
+      "if [ -f /app/eclipse/plugins/org.eclipse.epp.package.committers_*/eclipse256.png ] ; then cp -p /app/eclipse/plugins/org.eclipse.epp.package.committers_*/eclipse256.png /app/share/icons/hicolor/256x256/apps/org.eclipse.Committers.png ; fi",
```

Comment:
These icons are always available? We need at least 128 & 64px.

Reply:
Hmm, annoyingly no: https://git.eclipse.org/c/epp/org.eclipse.epp.packages.git/tree/packages/org.eclipse.epp.package.committers

This particular package has 16, 32, 48 and 256.

Reply:
In such case, the icon needs to be downscaled from 256 to the missing sizes and added then. Hopefully, we will have something that automatically does that in the future.

Reply:
Okay I can do that. Hopefully I can get upstream to ship more icons for the next release

Reply:
Fixed in 941db15

```
[mbooth@thinkpad-p50 flathub-java]$ flatpak run --command=bash org.eclipse.Java -c "find /app/share/icons"
/app/share/icons
/app/share/icons/hicolor
/app/share/icons/hicolor/128x128
/app/share/icons/hicolor/128x128/apps
/app/share/icons/hicolor/128x128/apps/org.eclipse.Java.png
/app/share/icons/hicolor/64x64
/app/share/icons/hicolor/64x64/apps
/app/share/icons/hicolor/64x64/apps/org.eclipse.Java.png
/app/share/icons/hicolor/32x32
/app/share/icons/hicolor/32x32/apps
/app/share/icons/hicolor/32x32/apps/org.eclipse.Java.png
/app/share/icons/hicolor/256x256
/app/share/icons/hicolor/256x256/apps
/app/share/icons/hicolor/256x256/apps/org.eclipse.Java.png
```